merlin-server/sql/merlin/tables/plan_snapshot.sql

```diff
@@ -8,12 +8,26 @@ create table plan_snapshot(
   plan_id integer
     references plan
     on delete set null,
-
   revision integer not null,
-  name text not null,
-  duration interval not null,
-  start_time timestamptz not null
+
+  snapshot_name text null,
+  taken_by text null,
```

Comment:
```suggestion
  snapshot_name text,
  taken_by text,
```

According to [postgres docs](https://www.postgresql.org/docs/current/sql-createtable.html):
"This [null] clause is only provided for compatibility with non-standard SQL databases. Its use is discouraged in new applications."

Reply:
`null` has been removed from the new columns 👍